### PR TITLE
New ioctl command for node reward and unit test

### PR DESCRIPTION
**Pull request by RuneAkko**

### CODE REVIEW

File: ioctl/client.go [hunk 1]
```diff
@@ -48,6 +48,8 @@ type (
 		// doing
 		GetAddress(in string) (string, error)
 		// doing
+		Address(in string) (string, error)
```
Comment (Frankonly) [inline ioctl/client.go]: What's the difference bewteen "GetAddress()" and "Address()" ?

Comment (RuneAkko) [reply ioctl/client.go]: all comes from iotcl/util.go.`GetAddress():GetAddress get address from address or alias or context`,`Address():Address returns the address corresponding to alias. if 'in' is an IoTeX address, returns 'in'`.

File: go.mod [hunk 2]
```diff
@@ -3,6 +3,7 @@ module github.com/iotexproject/iotex-core
 go 1.13
 
 require (
+	bou.ke/monkey v1.0.2
```
Comment (yutongp) [inline go.mod]: i don't see this dependence used anywhere. if we don't need it, please remove

File: ioctl/output/format.go [hunk 3]
```diff
@@ -73,15 +73,21 @@ const (
 
 // Output is used for format output
 type Output struct {
-	MessageType MessageType `json:"messageType"`
-	Message     Message     `json:"message"`
+	MessageType            MessageType            `json:"messageType"`
+	Message                Message                `json:"message"`
+	MessageWithTranslation MessageWithTranslation `json:"MessageWithTranslation"`
```
Comment (yutongp) [inline ioctl/output/format.go]: json:"messageWithTranslation

**Pull request merged**